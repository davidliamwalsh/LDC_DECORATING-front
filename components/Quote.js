import { Component } from 'react'
import Link from 'next/link'

class Quote extends Component {
  render () {
    return <>
      <div className="c-quote">
        <div className="c-quote__item">
          <p className="c-quote__item--text">FOR A FREE, NO OBLIGATION QUOTE</p>
          <Link href='/contact'>
              <a className="c-quote__item--button">Send us a message</a>
          </Link>
        </div>
      </div>
    </>
  }
}

export default Quote