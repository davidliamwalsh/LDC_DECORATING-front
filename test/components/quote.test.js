import Quote from '../../components/Quote'

describe('<Quote />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <Quote />
    )

    expect(component).toMatchSnapshot()
  })
})