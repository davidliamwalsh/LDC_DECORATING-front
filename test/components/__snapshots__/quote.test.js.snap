// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Quote /> it renders correctly 1`] = `ShallowWrapper {}`;
